ccm_web/server/src/canvas/canvas.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { CanvasController } from './canvas.controller';
+
+describe('CanvasController', () => {
+  let controller: CanvasController;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [CanvasController],
+    }).compile();
+
+    controller = module.get<CanvasController>(CanvasController);
+  });
+
+  it('should be defined', () => {
+    expect(controller).toBeDefined();
+  });
+});
```

Comment:
I started looking into writing robust unit tests for the `CanvasModule`, and it seems like it will take me a while to get up to speed with `jest` and the NestJS testing approach. I did make some progress here (https://github.com/ssciolla/canvas-course-manager-next/commit/a17af78227d14ac9c40179ac7bbd29c07cd7cb37) on mocking Nest dependencies so I could focus on the specific method logic in the controller and server classes.

@pushyamig and @lsloan, what do you think? Should I take the extra time now to flesh these out and add tests for each method? Or should I continue to work on this in parallel and open as a separate issue/PR?

Reply:
Okay, @pushyamig, see issue #97. I am going to remove these `spec` files from the current PR for now.